pkg/storage/local/store.go

```diff
@@ -31,12 +45,14 @@ type Layer struct {
 type Rig struct {
 	ID                int     `json:"id"`
 	Original          bool    `json:"original"`
+	RendersCid        *string `json:"renders_cid" db:"renders_cid"`
 	PercentOriginal   float64 `json:"percent_original" db:"percent_original"`
 	PercentOriginal50 float64 `json:"percent_original_50" db:"percent_original_50"`
 	PercentOriginal75 float64 `json:"percent_original_75" db:"percent_original_75"`
 	PercentOriginal90 float64 `json:"percent_original_90" db:"percent_original_90"`
 	VIN               string  `json:"vin"`
 	Parts             []Part  `json:"parts"`
+	Deals             []Deal  `json:"deals"`
```

Comment:
More updates to model deals in our local data model that is written to our local SQLite db.